egs2/tedx_spanish_corpus_openslr67/asr1/local/create_data.py

```diff
@@ -0,0 +1,36 @@
+import glob
+
+
+def main(cmd=None):
+    # get transcriptions
+    transcriptions = {}
+    transcripts = open("downloads/tedx_spanish_corpus/files/TEDx_Spanish.transcription")
```

Comment:
the current procedure assumes the data is in `downloads`, but it might not be true if `db.sh` is set to other values. Please try to smooth the stages to make them flexible to different directories.